### PR TITLE
PMM-7405 Add check conn to external exporter/

### PR DESCRIPTION
**Pull request by askomorokhov**
https://jira.percona.com/browse/PMM-7405

FB: https://github.com/Percona-Lab/pmm-submodules/pull/1518

Ref: 
 - https://github.com/percona/pmm-admin/pull/135
 - https://github.com/percona/pmm-managed/pull/659
 - https://github.com/percona/pmm/pull/679
 - https://github.com/percona/pmm-agent/pull/231

### CODE REVIEW

**Review by JiriCtvrtka — APPROVED** [review]

File: connectionchecker/connection_checker.go [hunk 1]
```diff
@@ -206,3 +209,20 @@ func (cc *ConnectionChecker) checkProxySQLConnection(ctx context.Context, dsn st
 
 	return &res
 }
+
+func (cc *ConnectionChecker) checkExternalConnection(ctx context.Context, uri string) *agentpb.CheckConnectionResponse {
+	var res agentpb.CheckConnectionResponse
+
+	resp, err := http.Get(uri)
```
Comment (percona-csalguero) [inline connectionchecker/connection_checker.go:216]: Does it make sense to use the context like you did [here](https://github.com/percona/pmm-managed/pull/659/files#diff-efe2a6cfe03bd019245b4fa72eb3a366a044451536644d616929aff48cc0da91R179)?

**Review by percona-csalguero — CHANGES_REQUESTED** [review]

File: connectionchecker/connection_checker.go (hunk 1 above)
Comment (askomorokhov) [reply connectionchecker/connection_checker.go:216]: Yeah, tricky point. Chenged.

**Review by percona-csalguero — APPROVED** [review]

File: connectionchecker/connection_checker.go (hunk 1 above)
Comment (BupycHuk) [inline connectionchecker/connection_checker.go:216]: maybe it's better to use `NewRequestWithContext` and have some timeout here. what do you think?

**Review by BupycHuk — CHANGES_REQUESTED** [review]

File: connectionchecker/connection_checker.go [hunk 2]
```diff
@@ -206,3 +210,26 @@ func (cc *ConnectionChecker) checkProxySQLConnection(ctx context.Context, dsn st
 
 	return &res
 }
+
+func (cc *ConnectionChecker) checkExternalConnection(ctx context.Context, uri string) *agentpb.CheckConnectionResponse {
+	var res agentpb.CheckConnectionResponse
+
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
```
Comment (BupycHuk) [inline connectionchecker/connection_checker.go:217]: let's just use context coming in args.
As far as I see we pass timeout here https://github.com/percona/pmm-agent/pull/231/files#diff-6eb9fa169fdb33f5cf7bf4a8267eef04668d3cee3ccb099593589a30067c141aR63

**Review by BupycHuk — APPROVED** [review]

**Pull request merged**